blocks/library/cover-image/index.js

```diff
@@ -133,11 +133,12 @@ registerBlockType( 'core/cover-image', {
 			</BlockControls>,
 			<InspectorControls key="inspector">
 				<h2>{ __( 'Cover Image Settings' ) }</h2>
```

Comment:
We can hide the `InspectorControls`, there is no need to show the settings headline.

Reply:
Done